build-system/travis.js

```diff
@@ -28,99 +28,124 @@ const {red, cyan} = colors;
+  return isTravisBuild() && process.env.TRAVIS_EVENT_TYPE === 'pull_request';
+}
 
 /**
  * Returns true if this is a Travis Push build.
  * @return {boolean}
  */
-exports.isTravisPushBuild = function() {
-  return exports.isTravisBuild() && process.env.TRAVIS_EVENT_TYPE === 'push';
-};
+function isTravisPushBuild() {
+  return isTravisBuild() && process.env.TRAVIS_EVENT_TYPE === 'push';
+}
 
 /**
  * Returns the build number of the ongoing Travis build.
  * @return {string}
  */
-exports.travisBuildNumber = function() {
-  if (!exports.isTravisBuild()) {
+function travisBuildNumber() {
+  if (!isTravisBuild()) {
     log(
       red('ERROR:'),
       'This is not a Travis build. Cannot get',
       cyan('process.env.TRAVIS_BUILD_NUMBER') + '.'
     );
   }
   return process.env.TRAVIS_BUILD_NUMBER;
-};
+}
 
 /**
  * Returns the job number of the ongoing Travis build.
  * @return {string}
  */
-exports.travisJobNumber = function() {
-  if (!exports.isTravisBuild()) {
+function travisJobNumber() {
+  if (!isTravisBuild()) {
     log(
       red('ERROR:'),
       'This is not a Travis build. Cannot get',
       cyan('process.env.TRAVIS_JOB_NUMBER') + '.'
     );
   }
   return process.env.TRAVIS_JOB_NUMBER;
-};
+}
 
 /**
  * Returns the repo slug associated with the ongoing Travis build.
  * @return {string}
  */
-exports.travisRepoSlug = function() {
-  if (!exports.isTravisBuild()) {
+function travisRepoSlug() {
+  if (!isTravisBuild()) {
     log(
       red('ERROR:'),
       'This is not a Travis build. Cannot get',
       cyan('process.env.TRAVIS_REPO_SLUG') + '.'
     );
   }
   return process.env.TRAVIS_REPO_SLUG;
-};
+}
 
 /**
  * Returns the commit SHA being tested by the ongoing Travis PR build.
  * @return {string}
  */
-exports.travisPullRequestSha = function() {
-  if (!exports.isTravisPullRequestBuild()) {
+function travisPullRequestSha() {
+  if (!isTravisPullRequestBuild()) {
     log(
       red('ERROR:'),
       'This is not a Travis PR build. Cannot get',
       cyan('process.env.TRAVIS_PULL_REQUEST_SHA') + '.'
     );
   }
   return process.env.TRAVIS_PULL_REQUEST_SHA;
-};
+}
 
 /**
  * Returns the name of the branch being tested by the ongoing Travis PR build.
  * @return {string}
  */
-exports.travisPullRequestBranch = function() {
-  if (!exports.isTravisPullRequestBuild()) {
+function travisPullRequestBranch() {
+  if (!isTravisPullRequestBuild()) {
     log(
       red('ERROR:'),
       'This is not a Travis PR build. Cannot get',
       cyan('process.env.TRAVIS_PULL_REQUEST_BRANCH') + '.'
     );
   }
   return process.env.TRAVIS_PULL_REQUEST_BRANCH;
+}
+
+/**
+ * Returns the commit SHA being tested by the ongoing Travis build.
+ * @return {string}
+ */
+function travisCommitSha() {
+  if (!isTravisBuild()) {
+    log(
+      red('ERROR:'),
+      'This is not a Travis build. Cannot get',
+      cyan('process.env.TRAVIS_COMMIT') + '.'
+    );
+  }
+  return process.env.TRAVIS_COMMIT;
```

Comment:
What exactly is this? Would this be different from `TRAVIS_PULL_REQUEST_SHA`? The docs don't explain it that well